content/_index.md

```diff
@@ -1,23 +1,80 @@
 ---
 title: Home
 ---
-{{< home/section title="What is it?" iconclass="fas fa-2x fa-power-off">}}
-GitBOM is...
+{{< home/section title="What?" iconclass="fas fa-2x fa-power-off">}}
+GitBOM is neither [git](/glossary/#git) nor an [SBOM](/resources/glossay/#sbom).
+<!-- 
+GitBOM is an ultra simple scheme for [primary build tools](/resources/glossay/#primary-build-tools) to embed into the artifact being built a
+short simple verifiable reference to complete artifact tree of that artifact with absolute precision all the way down to the source code. -->
+
+GitBOM is an ultra simple scheme for a [build tool](/glossary/build_tool) to:
+1. Construct a complete [artifact tree](/glossary/artifact_tree) with absolute precision all the way down to the individual source code files incorporated into the [artifact](/glossary/artifact) being built.
+2. Embed an id for that artifact tree into the artifact being built.
+
+in a language-heterogenous environment with zero developer effort, involvement, or awareness.
+
+- GitBOM is not an SBOM, though it is designed to complement existing SBOM efforts such as [SPDX](https://spdx.dev/), [CycloneDX](https://cyclonedx.org/), or [SWID](https://nvd.nist.gov/products/swid).
+- GitBOM is not a version control system. It is designed to co-exist with them, but not require them.
+
+GitBOM applies the [Unix Philosophy](https://en.wikipedia.org/wiki/Unix_philosophy). It does one thing well: constructing [artifact trees](/glossary/artifact_tree).
+
+<!-- GitBOM borrows the use of [merkle tree](/resources/glossay/#merkle-tree) and the [gitref](/resources/glossay/#merkle-tree) from [git](/resources/glossay/#mgit) to construct a [verifiable bare artifact tree](/resources/glossay/#verifiable-bare-artifact-tree) in a manner suitable for [primary build tools](/resources/glossay/#primary-build-tools) to compute the GitBOM for an artifact and embed a unqiue, immutable, verifiable GitBOM id in the artifact being built. 
+in language-heterogenuous environments with zero-end-user-effort. -->
 
-* Lorem Ipsum1
-* Lorem Ipsum2
 {{< /home/section >}}
+<!-- 
+{{< home/section title="GitBOM is not:" iconclass="fas fa-2x fa-power-off">}}
+1. Not a system for build reproducability, but it does provide information that is useful for that.
+2. Not a version control system, though it is designed to co-exist with them.
+3. Not an SBOM, though it is designed to complement them.
+4. Not a globally unique software identifier (SWID).
+5. Not reliant on any particular packaging or distribution mechanism, either for artifacts or for artifact identity graphs).
+{{< /home/section >}} -->
+
+
+{{< home/section title="Why?" >}}
```

Comment:
I've left a few specific suggestions below, but my gut says this still isn't the right impact. WDYT about the "WHY" section being a one-paragraph explanation of how GitBOM _would have helped_ identify log4j?

Something like this...

> By correlating every piece of software to a verifiable, complete, and concise artifact tree, GitBOM would have enabled enterprises to easily and independently identify which networked components (both open source and proprietary) contained the `log4j` library. GitBOM would allow an organization to reliably cross-reference all software they consume against CVE databases.

Reply:
Ah... interesting... more narrative :)  Will see what I can do there :)